src/site/content/en/patterns/image-demo-example/assets/sample.html

```diff
@@ -0,0 +1,3 @@
+<figure>
```

Comment:
This file seems unused because you load "body.html".

Reply:
This is a sample that gets displayed in one of the assets tab. It is independent of the image demo.

Reply:
Right, but it seems like it's purpose is to match the content that is in the Markdown, this is the same demo. Is that correct? Should it just be body.html?

Reply:
Not really, they should be independent. There are 4 types of demos:

Allow demo to be built by:
+ Providing html, js and css snippet
+ Providing a complete html page
+ Providing an external, frameable demo url
+ Providing a static image or gif

See here:
https://docs.google.com/document/d/1jpTAHkuwg4XDJ2pvHKNuRxywTk6-dCgBgdriMqPteAY/edit?usp=sharing

Only in the 1st case the body.html is identical with the .../demo.html url content.

The purpose of image demo is that you provide an image illustrating the concept described in the pattern, e.g. a screencast of devtools to illustrate assets loading.